src/common/target.h

```diff
@@ -87,6 +87,7 @@ typedef enum {
     TGT_C65,
     TGT_CX16,
     TGT_SYM1,
+    TGT_KIM1,                   // Added at end so as not to shift existing entries
```

Comment:
It may be better to not suggest adding this comment to the common codebase, but to explain this implementation decision in the PR description.